Sources/Purchasing/OfferingsManager.swift

```diff
@@ -171,6 +191,20 @@ private extension OfferingsManager {
 
 }
 
+extension OfferingsManager {
+
+    /// Determines the behavior when products in an `Offering` are not found
+    internal enum FetchPolicy {
+
+        case ignoreNotFoundProducts
+        case failIfProductsAreMissing
```

Comment:
Hmm do we have plans to add other policies here? Just wondering whether it would be better to just make this a boolean instead of an enum.

Reply:
Even if we don't, this allows us to document the behavior in the name. As for performance, this has essentially the same representation as a boolean.
The advantage over a `Bool` is that we don't have to wonder in every place what `true` or `false` means, and that's much more error prone.